sql_generators/glean_usage/templates/clients_last_seen_joined.metadata.yaml

```diff
@@ -6,6 +6,11 @@ description: |-
   Join on baseline and metrics views, but uses a LEFT JOIN
   such that a given client will not appear in the result if
   it is only represented in the metrics-based table.
+
+  NOTE FENIX ONLY:
```

Comment:
```suggestion
```

I believe this could happen for any app. For now we have seen it only on fenix (though I thought at some point this also happened on iOS)